### ISSUE TITLE
GSSOC EXT NOTICE: Improper Labeling in the Project

### ISSUE THREAD

**Issue by Meetjain1** [Soujanya2004/wanderlust-2024#24]
Dear Project Admin @Soujanya2004 

It has come to our attention that your repository not applying the necessary labels to issues and merged PRs, which is affecting contributor scores on the leaderboard.

Please ensure you use the correct labels:

- gssoc-ext
- level1, level2, level3 
- hacktoberfest-accepted (only if your repository have Hacktoberfest topic in description)

*NOTE:* Level labels should not have any dashes between them. Spaces are allowed, but no dashes should be used.

Kindly review and update the labels on your issues and PRs to ensure fairness for all contributors. Failure to address this may result in your project being flagged.

Thank you,
Core Team
GSSoC '24 Extended

**Comment by Meetjain1** [comment]
@Soujanya2004

**Comment by Soujanya2004** [comment]
@Meetjain1 I have corrected it, And I am not having project mentor for this project please check

**Comment by Meetjain1** [comment]
> @Meetjain1 I have corrected it, And I am not having project mentor for this project please check

But I cant see any labels on merged prs @Soujanya2004 , Sure I will check this

**Comment by Meetjain1** [comment]
@Soujanya2004 Your project is assigned to 3 mentors check PA and mentors sheet

**Comment by Soujanya2004** [comment]
@Meetjain1 all 4 are not managing the project, any suggestions

**Comment by Meetjain1** [comment]
> @Meetjain1 all 3 are not managing the project, any suggestions

did you contact them ?

**Comment by Soujanya2004** [comment]
I couldn't find the list @Meetjain1

**Comment by Meetjain1** [comment]
> I couldn't find the list @Meetjain1

Its already shared on discord long time ago , you can also find it here [link](https://github.com/orgs/GSSoC24/discussions/175)

**Comment by Meetjain1** [comment]
@Soujanya2004 also add correct labels soon many some contributors are facing issues because of this

**Comment by Soujanya2004** [comment]
@Meetjain1 i have fixed it please check once

**Comment by Meetjain1** [comment]
@Soujanya2004 "gssoc-ext" logo is not there !

**Comment by Soujanya2004** [comment]
@Meetjain1 should I add labels to pr and issues both?

**Comment by Soujanya2004** [comment]
@Meetjain1 http://www.linkedin.com/in/muhd-adnan-mohsin-7b01b8252  mentor profile is not found for contacting

**Comment by Meetjain1** [comment]
> @Meetjain1 should I add labels to pr and issues both?

Yes on both so that contributors didnt get confused , labels on prs is must as points are fetched from merged prs.

**Comment by Soujanya2004** [comment]
@Meetjain1  now its proper ryt?

**Comment by Meetjain1** [comment]
> @Meetjain1 now its proper ryt?

yes perfect !

**Comment by Meetjain1** [comment]
> @Meetjain1 http://www.linkedin.com/in/muhd-adnan-mohsin-7b01b8252 mentor profile is not found for contacting

can you try on discord ? I will also find and give update you If I forgot please remind me again

**Comment by Soujanya2004** [comment]
@Meetjain1 i am not able to find, can you help out

**Comment by Meetjain1** [comment]
> @Meetjain1 i am not able to find, can you help out

I will check and update you soon @Soujanya2004

**Comment by Meetjain1** [comment]
@AdnanMohsin21 you are assigned as a mentor in this project please contact PA @Soujanya2004 for further steps .

**Comment by Meetjain1** [comment]
@Soujanya2004 if he doesn't respond you can contact him via email adnanenigmatic1@gmail.com .

**Comment by Soujanya2004** [comment]
@Meetjain1 sure, thanks

**Comment by Soujanya2004** [comment]
@Meetjain1 another doubt, should i add assignees for pr also or is it only for issue raised

**Comment by Meetjain1** [comment]
> @Meetjain1 another doubt, should i add assignees for pr also or is it only for issue raised

Add on prs also
